File: 02_Programming_Fundamentals/02_Programming_Fundamentals_with_C#/02_Intro_And_Basic_Syntax_Exercise/10_RageExpenses.cs
using System;

public class RageExpenses
{
	public static void Main()
	{
		int LostGames = int.Parse(Console.ReadLine());
		float PriceHeadset = float.Parse(Console.ReadLine());
		float PriceMouse = float.Parse(Console.ReadLine());
		float PriceKeyboard = float.Parse(Console.ReadLine());
		float PriceDisplay = float.Parse(Console.ReadLine());
		
      	int counterEachSecondGame = 0;
		int counterEachThirdGame = 0;
		
		int counterHeadset = 0;
		int counterMouse = 0;
		int counterKeyboard= 0;
		int counterDisplay = 0;
	
		for(int i = 1; i <= LostGames; i++)
		{
			counterEachSecondGame++;
			counterEachThirdGame++;

			if(counterEachSecondGame == 2)
			{
				counterHeadset++;
			}
			if(counterEachThirdGame == 3)
			{
				counterMouse++;
				if(counterEachSecondGame == 2)
				{
					counterKeyboard++;
					if(counterKeyboard % 2 == 0)
					{
						counterDisplay++;
					}
				}
			}
			
			if (counterEachSecondGame == 2) { counterEachSecondGame = 0; }
			if (counterEachThirdGame == 3) { counterEachThirdGame = 0; }
		}
		
		double sum = PriceHeadset * counterHeadset + PriceMouse * counterMouse + PriceKeyboard * counterKeyboard + PriceDisplay * counterDisplay;
		Console.WriteLine("Rage expenses: {0:F2} lv.", sum);
	}
	
}

File: 02_Programming_Fundamentals/04_Programming_Fundamentals_PHP/02_Intro_And_Basic_Syntax_Exercise/10_Rage_Expenses.php
<?php 

$LostGames = intval(readline());
$PriceHeadset = floatval(readline());
$PriceMouse = floatval(readline());
$PriceKeyboard = floatval(readline());
$PriceDisplay = floatval(readline());

$counterEachSecondGame = 0;
$counterEachThirdGame = 0;

$counterHeadset = 0;
$counterMouse = 0;
$counterKeyboard= 0;
$counterDisplay = 0;

for($i = 1; $i <= $LostGames; $i++) {
  $counterEachSecondGame++;
  $counterEachThirdGame++;

  if($counterEachSecondGame == 2) {
    $counterHeadset++;
  }
  if($counterEachThirdGame == 3) {
  $counterMouse++;

    if($counterEachSecondGame == 2) {
    $counterKeyboard++;

      if($counterKeyboard % 2 == 0) {
        $counterDisplay++;
      }
    }
  }

if ($counterEachSecondGame == 2) { $counterEachSecondGame = 0; }
if ($counterEachThirdGame == 3) { $counterEachThirdGame = 0; }
}

$sum = $PriceHeadset * $counterHeadset + $PriceMouse * $counterMouse + $PriceKeyboard * $counterKeyboard + $PriceDisplay * $counterDisplay;

echo "Rage expenses: " . number_format($sum, 2, '.', '') . " lv.";

?>